android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java

```diff
@@ -106,6 +109,12 @@ public void onCreate() {
         }
         refWatcher = LeakCanary.install(this);
 
+        //Initialize Cache
+        File httpCacheDirectory = new File(getCacheDir(), "picasso-cache");
+        Cache cache = new Cache(httpCacheDirectory, 15 * 1024 * 1024);
+
```

Comment:
This is disk space @arp95c`cacheDir` -> cache directory

If it would have been allocating memory, it would be in memory cache making all saved objects clear after app reload, so wouldn't make sense